mysql-test/suite/innodb/t/bug76142.test

```diff
@@ -0,0 +1,42 @@
+--echo #
+--echo # Bug 76142: InnoDB tablespace import fails when importing table w/ different data directory
+--echo #
```

Comment:
can we move this to existing test import_tablespace_schema_missmatch.test?

Reply:
moved.